Description: CFLAGS gets exported in debian/rules, but eventually overwritten  #967239
Forwarded: not-needed
--- a/src/Makefile
+++ b/src/Makefile
@@ -200,8 +200,10 @@
 
 LIBS = $(XLIBS) -lm $(ADDLIBS) 
 
+DPKG_CFLAGS := $(CFLAGS) # defined by dpkg-buildflags
 CFLAGS = $(FPIC) $(WALL) -DPLOTICUS -D$(PLATFORM) -DTDH_NOREC $(ZFLAG) $(LOCALE_FLAG) \
-  $(NOGD) $(NOX11) $(NOPS) $(NOSVG) $(NOSWF) $(XINCLUDEDIR) $(MINGH) 
+  $(NOGD) $(NOX11) $(NOPS) $(NOSVG) $(NOSWF) $(XINCLUDEDIR) $(MINGH) \
+  $(DPKG_CFLAGS)
 
 OBJ = proc_annotate.o proc_areadef.o proc_axis.o proc_bars.o proc_rect.o proc_breakaxis.o \
   proc_categories.o proc_curvefit.o proc_settings.o proc_drawcommands.o \
